workspaces/preferences.md

```diff
@@ -68,8 +65,6 @@ Please note that:
 - You must enable native notifications for each browser on which you run Coder
   Enterprise
 
-You can check whether notifications are enabled and working by click **Test**.
```

Comment:
PS: there was a type here (click -> clicking), if we want to backport that to current versions.